source/install-by-copying-files/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Install by copying files
+weight: 10
+---
+
+# Install by copying files
+
+You can install GOV.UK Frontend by copying our CSS and JavaScript files into your project.
+
+<%= warning_text('We recommend <a href="/installing-with-npm/">installing with Node.js package manager (npm)</a> instead.
+') %>
+
+## What you cannot do
+
+If you install this way, you will not be able to:
+
+- change settings, for example you will not be able to override colours or set your own font
+- use the Nunjucks macros from the [Design System website](https://design-system.service.gov.uk/)
+- import an individual component’s CSS or JavaScript
+- use GOV.UK Frontend's colours or mixins in your custom code
+
+## Copy the files
+
+1. Download the [`dist` folder and its subfolders](https://github.com/alphagov/govuk-frontend/tree/master/dist) from the GOV.UK Frontend GitHub repo.
+2. Copy the 2 `.css` files to `<YOUR-APP>/stylesheets`.
+3. Copy the `.js` file to `<YOUR-APP>/javascript`.
+4. Copy the `assets` folder to the root of your project’s public folder, so that for example `<YOUR-SITE-URL>/assets/govuk-logotype-crown.png` will work in your users’ browsers.
+
+## Check it works
+
+1. Create a page in your project using the following HTML:
+
+    ```html
+    <!DOCTYPE html>
+    <html lang="en" class="govuk-template ">
+      <head>
+        <title>Example if you’ve installed GOV.UK Frontend by copying files</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+        <!--[if !IE 8]><!-->
+          <link rel="stylesheet" href="/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css">
+        <!--<![endif]-->
+        <!--[if IE 8]>
+          <link rel="stylesheet" href="/stylesheets/govuk-frontend-ie8-<VERSION-NUMBER>.min.css">
+        <![endif]-->
+      </head>
+      <body class="govuk-template__body ">
+        <script>
+          document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+        </script>
+        <!-- component HTML -->
+        <script src="/javascript/govuk-frontend-<VERSION-NUMBER>.min.js"></script>
+        <script>
+          window.GOVUKFrontend.initAll()
+        </script>
+      </body>
+    </html>
+    ```
+
+    In your live application, you should use the [page template](https://design-system.service.gov.uk/styles/page-template/) instead.
```

Comment:
Or even if they just want to get a complete page with header and footer for testing.

Reply:
Ah yes - I played around with actually replacing this first step with 'copy the page template HTML'. But I think it makes the following steps a bit more complicated, so I think it might be best not to suggest users use the template at this point?

Reply:
Thanks @m-green. I don't think we should suggest they use the template, but I wonder if instead of 'live application' this could say something about getting a full page, even if it's for testing.
> To get the full page template with header and footer components, you should use ...

Reply:
We could, but then it might sound like an alternate option in this step - and then they'll hit trouble from step 2 onwards, because they'll need to do more with the full page template to get it working (that we don't take them through here).
I might be misunderstanding what you mean though, so let me know!

Reply:
Thanks @m-green. I see your point about it possibly looking like an alternative step. Let's go with your suggestion and see if it comes up on support.